### DISCUSSION TITLE
getting issue with insertion of data

### DISCUSSION THREAD

**Discussion by taibarehm** [mkleehammer/pyodbc#1281] [Q&A]
Please first make sure you have looked at:

* Documentation: https://github.com/mkleehammer/pyodbc/wiki
* Other issues

### Environment

To diagnose, we usually need to know the following, including version numbers.  On Windows, be
sure to specify 32-bit Python or 64-bit:

- Python: 
- pyodbc: 
- OS: 
- DB:
- driver:

### Issue

Often it is easiest to describe your issue as "expected behavior" and "observed behavior".

**Comment by taibarehm** [comment]
5     cursor.execute(SQLCommand,i.timestamp,str(i.title),float(i.price),str(i.market),float(i.surface),str(i.location),str(i.remote_support),str(i.lighting),str(i.advertiser_type),str(i.description),int(i.no_of_rooms),str(i.form_of_property),str(i.url),str(i.is_for_sale),str(i.posting_id))
      6     connection.commit()
      7 connection.close()

DataError: ('22003', '[22003] [Microsoft][ODBC SQL Server Driver]Numeric value out of range (0) (SQLExecDirectW)')

**Comment by v-chojas** [comment]
The information you were asked for when creating the issue is important, please fill it in.

**Comment by gordthompson** [comment]
Also, please provide a [minimal reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) that shows your issue.
